app/lib/account/models.dart

```diff
@@ -21,6 +21,11 @@ class User extends db.ExpandoModel {
 
   @db.DateTimeProperty()
   DateTime created;
+
+  @db.BoolProperty()
+  bool deletedFlag;
+
+  bool get isDeleted => deletedFlag == true;
```

Comment:
Note: I think this is a much better pattern than having to use `isDeleted == true` in all the other places. If you like it then we should migrate the `Package.isDiscontinued` and similar flags to follow the same pattern.

Reply:
Regardless of what we do, this is non-trivial to understand. We need to explain why we have this property and what it means.

Maybe, something like:
>   `isDeleted` is set when a user account is deleted. When this happens user-data such as preferences are purged.
However, we retain the user entity if and only if the user has uploaded packages or appears in the history by other means. This is to ensure that we can see:
 (A) who uploaded a package, and,
 (B) who granted the permissions that allowed said package to be uploaded.

If we don't write this down, we have zero chance of remembering exactly why we retained user entities with an `isDeleted` property.